docs/v1/user-story

```diff
@@ -0,0 +1,124 @@
+A Year In The Life Of A Kubernetes Service Developer
+
+0. Today
```

Comment:
same here regarding the markdown header format. maybe consider using `##` as a subheader?